### PR TITLE
Automatically publish companies that meet criteria

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/ED-938)

It's not longer possible to manually publish companies, so I've removed code related to doing that.

### CODE REVIEW

File: company/admin.py [hunk 1]
```diff
@@ -65,20 +17,7 @@ class CompanyAdmin(admin.ModelAdmin):
...
-            url(
-                r'^publish/$',
-                self.admin_site.admin_view(
-                    PublishByCompanyHouseNumberView.as_view()
-                ),
-                name="company_company_publish"
-            ),
-        ]
-        return additional_urls + urls
+    readonly_fields = ('created', 'modified', 'is_published')
```
Comment (monikasulik) [inline company/admin.py]: Is is_published definitely a field that should not be possible to change? What if somebody puts up something offensive and we want to unpublish them?

File: company/models.py [hunk 2]
```diff
@@ -59,7 +59,14 @@ class Company(TimeStampedModel):
         blank=True,
         null=True,
     )
-    is_published = models.BooleanField(default=False)
+    is_published = models.BooleanField(
+        default=False,
+        help_text=(
+            'Companies are automatically published based on profile '
+            'completeness - they must have description or summary, be '
+            'verified, and have en email address.'
```
Comment (monikasulik) [inline company/models.py]: typo: an rather than en

File: company/admin.py (hunk 1 above)
Comment (richtier) [inline company/admin.py]: the code in signals.py will automatically set `is_published` back to `True` if we put it to `False` becasue the criteria it checks for is still met.

If we want to 'unpublish' we can untick 'is_verified', which will make the criteria no longer met, and the code will automatically set `is_published` to `False.`

Comment (monikasulik) [inline company/admin.py]: Ah, but then we lose the info that somebody received the letter and put the code in, but good point. I think maybe something we need to check with Alfie, but after that explanation no longer feel this is a merge stopper.

Comment (richtier) [inline company/admin.py]: cool. I ran it by him already, but we can add an explicit field if this becomes problematic. the field can be `is_hidden` or something,

**Pull request merged**